### ISSUE TITLE
General stats page

### ISSUE THREAD

**Issue by israellevin** [paket-core/web#5]
The general stats page will contain information on the number of packages in the system, the distance packages traveled, the number of tokens, etc.

This is a low priority right now and we need to discuss it before starting work.

**Comment by VitaliyKvas** [comment]
What api to get all the packages?

**Comment by israellevin** [comment]
https://route.paket.global/ has a swagger page with descriptions of all calls. But please do not do this now - we still have the pilot to finish.
